content/blog/pulumi-aws-cloudcontrol-provider/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: "Announcing the General Availability of Pulumi’s AWS Cloud Control Provider (formerly AWS Native) "
+allow_long_title: true
+date: 2024-04-25T09:00:00-07:00
+draft: false
+meta_desc: >-
+    Pulumi announces the GA of its AWS Cloud Control Provider, offering Day 1 support for new AWS features and enhancing cloud management tools.
+meta_image: 
+authors:
+    - tejitha-raju
+    - matt-jeffryes
+tags:
+    - aws 
+    - provider 
+    - cloudcontrol 
+---
+We're excited to announce the general availability of the new [AWS Cloud Control Provider] for Pulumi, previously AWS Native. AWS is one of the most-used cloud providers across the Pulumi ecosystem, and with this launch, we are focused on delivering Day 1 availability and faster access to new AWS capabilities to all Pulumi users. As part of this launch, we are also renaming the provider from “AWS Native” to "AWS Cloud Control Provider," based on customer feedback during the preview phase, to provide users with a broader range of advanced cloud management tools exposed by the AWS Cloud Control API. This provider includes the following features and benefits, 
+
+* __Day 1 support for AWS Resources__: Provides full coverage of the [AWS Cloud Control API], typically on the day of the launch, ensuring users can adopt new features immediately. 
+* __Complementary Pulumi Experience__: Works alongside Pulumi’s AWS Provider, enhancing Infrastructure as Code (IaC) projects with the latest AWS capabilities without requiring significant changes to existing setups. 
+* __Extended support for third-party resources__: Our platform offers enhanced flexibility and interoperability by supporting third-party resources available in the CloudFormation Registry, such as Atlassian, MongoDB, Datadog, and more.
+* __Easy CloudFormation to Pulumi Migration__: [cf2pulumi](https://www.pulumi.com/cf2pulumi/) provides the ability to migrate existing CloudFormation templates into Pulumi programs in your favorite language. 
+
+
+At Pulumi, we're dedicated to empowering our customers with the tools they need to innovate and thrive in the cloud. The launch of the AWS Cloud Control Provider represents our unwavering commitment to providing best-in-class solutions that simplify cloud management, streamline operations, and drive business success. The Pulumi AWS Cloud Control Provider can be used side-by-side with the standard  Pulumi AWS provider and nearly 200  additional Pulumi resource providers that cover a wide variety of cloud and SaaS platforms.  
+
+## Using Pulumi AWS Cloud Control Provider with the Pulumi AWS Provider
+
+Let’s walk through an example of using Pulumi AWS Cloud Control Provider alongside the [standard AWS Provider]. Here, we can see how the new AWS S3 Object Lambda feature can be used via the AWS Cloud Control provider, with access to the full API defined by the S3 team at AWS,
```

Comment:
The example below doesn't appear to use AWSCC at all?  And doesn't do what the prose above says?